jedi_vim.py

```diff
@@ -640,7 +640,7 @@ def new_buffer(path, options='', using_tagstack=False):
         if user_split_option not in split_options:
             print('g:jedi#use_splits_not_buffers value is not correct, valid options are: %s' % ','.join(split_options.keys()))
         else:
-            vim_command(split_options[user_split_option] + " %s" % path)
+            vim_command(split_options[user_split_option] + " %s" % escape_file_path(path))
```

Comment:
I don't think so. Seems like the `path` is escaped elsewhere.

Can you hint, what test exactly is needed here?